src/System.Private.CoreLib/shared/System/Math.cs

```diff
@@ -26,16 +26,6 @@ public static partial class Math
 
         public const double PI = 3.14159265358979323846;
 
-        private const int maxRoundingDigits = 15;
-
-        private static double doubleRoundLimit = 1e16d;
```

Comment:
Can this be changed to `const`? It is a constant.

Reply:
Already done that but will move this change to separate PR